src/node.ts

```diff
@@ -101,7 +102,11 @@ async function bundle(
       ...(isBadCommonJs(input) ? [(virtual as any)({"-": shimCommonJs(input, require)})] : []),
       importResolve(input, cacheRoot, packageRoot),
       nodeResolve({browser: true, rootDir: packageRoot}),
-      (commonjs as any)({esmExternals: true}),
+      json(),
```

Comment:
This is needed for CommonJS packages that `require("./package.json")` (which seems like a reasonable and easy thing for us to support).